doc/releases/luminous.rst

```diff
@@ -1,3 +1,157 @@
+v12.2.11 Luminous
+=================
+
+This is the eleventh bug fix release of the Luminous v12.2.x long term stable
+release series. We recommend that all users upgrade to this release. Please note
+the following precautions while upgrading.
+
+Notable Changes
+---------------
+
+* A previous release, v12.2.9, introduced the PG hard-limit patches which were
+  found to cause an issue in certain upgrade scenarios, and 12.2.10 release was
+  expedited to revert those patches. If you already successfully installed
+  v12.2.9, then you can upgrade to 12.2.11 directly. 12.2.10 users need no
+  special precaution.
+* This release fixes the pg log hard limit bug (https://tracker.ceph.com/issues/36686).
+  A flag called `pglog_hardlimit` has been introduced. It is off by default.
+  This flag enables the feature that limits the length of the pg log. Users
+  should run `ceph osd set pglog_hardlimit` after completely upgrading to
+  12.2.11. Once all the OSDs have this flag set, the length of the pg log will
```

Comment:
it's a single cluster-wide flag, so "once all the osds have this flag set" is confusing.

Reply:
Got it.